src/main/java/module/series/promotion/LeaguePromotionHandler.java

```diff
@@ -48,12 +43,14 @@ static class DownloadDetails {
      * @return boolean – true if promotion manager can be used, false otherwise.
      */
     public boolean isActive(int seriesId) {
-        List<Integer> supportedLeagues = HttpDataSubmitter.instance().fetchSupportedLeagues();
-        int[] activeWeeks = ModuleConfig.instance().getIntArray("PromotionStatus_ActiveWeeks", new int[] { 14, 15 });
-        int week = HOVerwaltung.instance().getModel().getBasics().getSpieltag();
-        return UserParameter.instance().promotionManagerTest ||
-                (Arrays.stream(activeWeeks).boxed().collect(Collectors.toList()).contains(week) &&
-                        supportedLeagues.contains(seriesId));
+        // TODO: fix HT-Server. As long as it is not working this feature will be disabled
+        return false;
+//        List<Integer> supportedLeagues = HttpDataSubmitter.instance().fetchSupportedLeagues();
```

Comment:
Ah sad. :-)

Is it because the HO server is not working any more?

Reply:
@tychobrailleur yes - the server is not working anymore. i remember some error messages concerning exceeded quotas or something like this.

Reply:
#1412

Reply:
> @tychobrailleur yes - the server is not working anymore. i remember some error messages concerning exceeded quotas or something like this.

Yes, the server that was being used was super-restricted in terms of storage, so not surprised.  As far as I remember, the server-side logic was written in Python by @akasolace, I cannot remember if this was stored in a git repo somewhere.